docs/tutorials/transferring-assets.md

```diff
@@ -73,7 +73,8 @@ You'll need the following information to set up a recurring transfer:
 * `startEpoch`: The number of the epoch in which you want the first transfer to be made. It will initiate at the end of that epoch.
 * `factor`: Written as a decimal less than 1.0. Factor is used to determine what portion of the full `amount` is transferred in each epoch. Think of it like a percentage, so the number you include, when multiplied by 100, will equal what percentage of the amount will be transferred each time. 
 
-If you do not have enough to cover each transfer and its fee, the transfer will automatically be cancelled.
+If you do not have enough to cover each transfer and its fee, the transfer will automatically be cancelled. The transfer will repeated indefinitely, unless you add the optional parameter to end the recurring transfer:
```

Comment:
```suggestion
If you do not have enough to cover each transfer and its fee, the transfer will automatically be cancelled. Otherwise, the transfer will repeated indefinitely, unless you add the optional parameter to end the recurring transfer:
```